src/components/todo/TodoModal.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+import { TextField, IconButton } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+interface TodoItem {
+  id: number;
+  text: string;
+  startTime: string;
+  endTime: string;
+}
+interface TodoModalProps {
+  open: boolean;
+  handleClose: () => void;
```

Comment:
```suggestion
  handleClose: VoidFunction;
```
함수가 인자를 받지 않고 아무것도 반환하지 않는것은 VoidFunction이라는 타입으로 쓸 수 있어요~

Reply:
팁 감사합니다~

Reply:
- ts 에러 : 'void' 형식은 'VoidFunction' 형식에 할당할 수 없습니다.ts(2322)	
	
- 해결책 : 	
	•	void와 VoidFunction: void는 함수가 반환하지 않는 값을 의미합니다. VoidFunction은 () => void의 별칭일 뿐, 타입에 대한 차이가 발생할 수 있습니다.
	•	해결 방법: () => void를 사용하여 타입을 명확히 정의하는 것이 좋습니다.
---

void -> VoidFunction으로 변경했더니 위와 같은 lint에러가 떠서 현재상황에서는 void로 유지할 수 밖에 없을 것 같습니다. 혹시  VoidFunction으로 변경하실 때 이런 ts에러가 나지 않으셨을까요?

Reply:
오 그런 lint 에러는 따로 보지 못했어요..! 저도 한번 찾아봐야겠네요.. 정보 감사합니다!